src/profiles.tsx

```diff
@@ -300,4 +301,25 @@ export const profiles = [
       </>
     ),
   },
+  {
+    name: 'Guliette Bernard',
```

Comment:
Show me what this looks like? Share a screenshot. She is not a bridesmaid.. Here is my spiel 

> Wasn’t Victoria’s sister a bridesmaid? 
No... Victoria did not kick Guliette - her middle sister - out of the wedding. Turmoil over the last 5 years has existed between the two. Guliette threatened to cancel at a bad time late in the game. The show went on. Guliette decided to lovingly apologize in a very sweet, creative ways that were not necessary but appreciated.

Reply:
That's my fault for not catching BM role. Peep the screenshots and let me know if I should keep it. 
![guliettePartyCard](https://user-images.githubusercontent.com/77205515/136475003-85391705-1f2f-44bf-8a7c-14e319967f2b.JPG)
![gulietteSoloCard](https://user-images.githubusercontent.com/77205515/136475011-52da31b1-4ac8-4ed3-9c10-c1273dd89776.JPG)

TLDR: Removing her role left an awkward card spacing so I assigned a new role to her that fit the best IMO

Reply:
![simplySister](https://user-images.githubusercontent.com/77205515/136476340-28074027-9644-4e7b-9419-54fa32f9a7eb.JPG)
![sisterOfThe](https://user-images.githubusercontent.com/77205515/136476342-12fd87ae-8dcd-46b5-986a-42d0b4a80c81.JPG)

1 or 2